cryptofeed/exchange/kraken_futures.py

```diff
@@ -246,15 +246,6 @@ async def _funding(self, msg: dict, pair: str, timestamp: float):
                                 next_funding_time=funding_time + interval,
                                 interval=interval,
                                 )
-        else:
```

Comment:
I can't comment on the exact line, as it hasn't changed, but above this we're referencing the `relative_funding_rate_prediction` and the `relative_funding_rate`. In the upstream, they're now using `funding_rate_prediction` and `funding_rate`. Should I also change that, or are we okay to stick to relative @tristan-murfitt-elw ?